kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/DeploymentUtils.java

```diff
@@ -235,6 +241,31 @@ public static void registryCredentialsSecret(String namespace) {
         }
     }
 
+    /**
+     * Collect cluster info.
+     *
```

Comment:
not the this PR, but I think it would be good to extend this to collect all the logs from all components we are interested in.  We could add kube labels to the deployments to make finding the right the pods easy.

Reply:
All logs from pods are included in the dump directory in a `logs.txt` file for each pod.